metricbeat/module/postgresql/_meta/config.yml

```diff
@@ -1,2 +1,21 @@
 - module: postgresql
+  #metricsets:
+  #  # Stats about every PostgreSQL database
```

Comment:
I would remove the comments for the metricsets. For these details we have the docs.